src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyState);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyState);
+  }
+
+  handleKeyState = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: `The last pressed key is [${event.key}]`,
+    });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          { pressedKey || 'Nothing was pressed yet'}
```

Comment:
Nice solution! It might be also useful to create a variable for "Nothing was pressed" message